resources/addresses/address.yml

```diff
@@ -20,3 +20,26 @@ get:
   responses:
     '200':
       $ref: responses/address.yml
+
+delete:
+  operationId: delete_address
+
+  summary: Deletes address with given id
+
+  description: >-
+    Deletes the details of an existing address. You need only supply
+    the unique customer identifier that was returned upon address creation.
+
+  tags:
+    - Addresses
+
+  parameters:
+    - in: path
+      name: id
+      required: true
+      schema:
+        $ref: 'attributes/adr_id.yml#/id'
+
+  responses:
+    '200':
+      $ref: responses/address.yml
```

Comment:
does the delete actually return?  We have some reason to believe that it doesn't based on working with this in webhooks.

Reply:
i dont see this file `responses/address.yml` in the repo or PR

Reply:
> i dont see this file `responses/address.yml` in the repo or PR

It's there. The path is relative: https://github.com/lob/hackathon-spec-v1/blob/main/resources/addresses/responses/address.yml

Reply:
We don't actually return the address when deleting it (presumably because doing so would make it hard to make the delete endpoint idempotent). The response for delete endpoints is different, returning the id and `deleted: true`. It's the same format for all of our `delete` (ahem, cancel) endpoints, so you might as well put that response in `shared/responses`.